src/components/listitems/ListItemNav.tsx

```diff
@@ -171,6 +178,14 @@ export const ListItemNav = ({
               />
             </View>
           )}
+          {paymentLogo && (
+            <View style={{ marginRight: IOListItemVisualParams.iconMargin }}>
+              <LogoPayment
+                name={paymentLogo}
+                size={IOSelectionListItemVisualParams.iconSize}
```

Comment:
Stick to the `IOListItemVisualParams` object, instead of importing the one related to **Selection** components
```suggestion
                size={IOListItemVisualParams.iconSize}
```

Reply:
Addressed into 48ff0bbd8c7d54c311814065e18c302e6c61a6a6